gapis/gfxapi/test/gfxapi_test_import/imported.go

```diff
@@ -1,49 +0,0 @@
-// Copyright (C) 2017 Google Inc.
```

Comment:
Sorry, just properly read the comment. Yeah, it was left over crap from the named imports and wasn't referenced.